en/developer_guide/other_database_type.rst

```diff
@@ -0,0 +1,50 @@
+Using other database type
+=========================
+
+Context
+-------
+
+Open orchestra use by default a MongoDb database. If you want to use another type of database like SQL,
+you need to do some work for now.
+
+Change object manager in configuration
+--------------------------------------
+
+By default, open orchestra use Doctrine ODM to manage object. Developers can define another document manager implementing
+``Doctrine\Common\Persistence\ObjectManager`` with a change in configuration
+
+.. code-block:: yaml
+
+  open_orchestra_base_bundle:
+      object_manager: namespace/of/objectManager
+
+Override mongo documents and repositories
+-----------------------------------------
+
+Some bundles have dependencies to mongo. To replace them, their is several solution depending of the type of dependencies.
+
+All the bundle need to be replace
```

Comment:
Not sure of what you want to tell. Maybe something simplier if it's the meaning: "Bundles needing to be replaced"